src/modules/31-filestore/components/FileStoreComponent/FileStoreComponent.tsx

```diff
@@ -78,7 +80,8 @@ const useFileStoreModal = ({ applySelected, fileUsage }: UseFileStoreModalProps)
   const commonProps = {
     isModalView: true,
     onNodeChange: handleSelectFile,
-    fileUsage
+    fileUsage,
+    handleSetIsUnsaved: setIsUnsaved
```

Comment:
This prop can remain as `setIsUnsaved`
```suggestion
    setIsUnsaved
```